Firestore/core/test/unit/api/load_bundle_task_test.cc

```diff
@@ -26,6 +26,20 @@
 namespace firebase {
 namespace firestore {
 namespace api {
+namespace {
+
+std::unique_ptr<util::Executor> CreateUserQueue() {
+  return util::Executor::CreateSerial("Testing User Queue");
```

Comment:
Nit: add a using declaration for `util::Executor`?

Reply:
Done.